### PR TITLE
Configure actual parameters for openapi

### PR DESCRIPTION
**Pull request by dsteinich**
Before making a pull request
----------------------------

- [ ] Make sure all tests run
- [ ] Update the changelog appropriately

Title
-----------
Brief description of changes. Reference the JIRA ticket if appropriate

Description
-----------
If no JIRA ticket is referenced, describe the changes made. Note anything that you want the reviewers to know while
reviewing your pull request

After making a pull request
---------------------------
- [ ] If appropriate, put the link to the PR in the JIRA ticket
- [ ] Assign someone to review unless the change is trivial

### CODE REVIEW

File: src/main/java/gov/usgs/wma/wqp/openapi/annotation/query/FullParameterList.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+//@SampleMedia
+//@Siteid
+//@SiteType
+//@Sorted
+//@StartDateHi
+//@StartDateLo
+//@Statecode
+//@SubjectTaxonomicName
+//@Within
+//@Zip
```
Comment (ssoper-usgs) [inline src/main/java/gov/usgs/wma/wqp/openapi/annotation/query/FullParameterList.java:38]: These are commented out - may or may not want to keep them

File: src/main/java/gov/usgs/wma/wqp/openapi/annotation/query/Zip.java [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+		description = "Should the data be compressed. Compression often greatly increases throughput, thus expediting the request.",
+		schema = @Schema(type = "string"),
+		examples = {
+				@ExampleObject(name="Yes", description="Compress the data.", value="yes"),
+				@ExampleObject(name="No", description="Do not compress the data.", value="no")
+				}
+		)
+public @interface Zip {
+
+}
```
Comment (ssoper-usgs) [inline src/main/java/gov/usgs/wma/wqp/openapi/annotation/query/Zip.java:30]: need a newline here

File: src/main/java/gov/usgs/wma/wqp/openapi/annotation/query/FullParameterList.java (hunk 1 above)
Comment (dsteinich) [reply src/main/java/gov/usgs/wma/wqp/openapi/annotation/query/FullParameterList.java]: my mistake, should have been un-commented.

File: src/main/java/gov/usgs/wma/wqp/openapi/model/post/PostParms.java [hunk 3]
```diff
@@ -0,0 +1,32 @@
+package gov.usgs.wma.wqp.openapi.model.post;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class PostParms extends MonitoringLocationPost {
```
Comment (mbucknell) [inline src/main/java/gov/usgs/wma/wqp/openapi/model/post/PostParms.java:5]: Really a nit. Elsewhere parameters is abbreviated as "params" vs "Parms" as it is here.

**Pull request merged**